libnavui-androidauto/src/main/java/com/mapbox/androidauto/car/navigation/ActiveGuidanceScreen.kt

```diff
@@ -71,13 +75,15 @@ class ActiveGuidanceScreen(
         }
     }
 
+    private var styleLoadedListener: OnStyleLoadedListener? = null
+
     private val surfaceListener = object : MapboxCarMapObserver {
 
         override fun onAttached(mapboxCarMapSurface: MapboxCarMapSurface) {
             super.onAttached(mapboxCarMapSurface)
             logAndroidAuto("ActiveGuidanceScreen loaded")
-            mapboxCarMapSurface.mapSurface.getMapboxMap().getStyle { style ->
-                placesLayerUtil.initializePlacesListOnMapLayer(style, carContext.resources)
+            styleLoadedListener = mapboxCarMapSurface.handleStyleOnAttached {
```

Comment:
```suggestion
            mapboxCarMapSurface.getStyle {
```

This one should not re-register routes observers

Reply:
it should, because a routes observer calls
```
mapboxCarMapSurface.getStyle {
            placesLayerUtil.updatePlacesListOnMapLayer(it, featureCollection)
        }
```

Reply:
Well, it doesn't really matter since the routesObserver is CopyOnWriteArraySet
```
private val routesObservers = CopyOnWriteArraySet<RoutesObserver>()
```